### PR TITLE
*: add TiDB basic features and update overview

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

- Add `basic-features.md`
- Delete `key-features.md`
- Update `overview.md`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3166
- Other reference link(s): Fix typos, descriptions and format for docs-cn in https://github.com/pingcap/docs-cn/pull/3784

### CODE REVIEW

File: overview.md [hunk 1]
```diff
@@ -1,51 +1,50 @@
...
 
-    - [kind](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/get-started#create-a-kubernetes-cluster-using-kind)
-    - [Minikube](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/get-started#create-a-kubernetes-cluster-using-minikube)
+- **Real-time HTAP**
 
-- [Binary Tarball Deployment](/production-deployment-from-binary-tarball.md): This guide describes how to deploy TiDB from a binary tarball in production. Guides for [development](/deploy-tidb-from-binary.md) and [testing](/test-deployment-from-binary-tarball.md) environments are also available.
+    TiDB provides two storage engines: [TiKV](https://tikv.org/), a row storage engine, and [TiFlash](https://docs.pingcap.com/tidb/v4.0/tiflash-overview), a column storage engine. TiFlash uses the Multi-Raft Learner protocol to replica data from TiKV in real time, ensuring that the data between the TiKV row storage engine and the TiFlash column storage engine are consistent. TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
```
Comment (ran-huang) [inline overview.md:24]:
```suggestion
    TiDB provides two storage engines: [TiKV](https://tikv.org/), a row-based storage engine, and [TiFlash](https://docs.pingcap.com/tidb/v4.0/tiflash-overview), a columnar storage engine. TiFlash uses the Multi-Raft Learner protocol to replicate data from TiKV in real time, ensuring that the data between the TiKV row-based storage engine and the TiFlash columnar storage engine are consistent. TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
```
Ref: https://docs.pingcap.com/tidb/dev/whats-new-in-tidb-4.0#storage-engine

File: overview.md [hunk 2]
```diff
@@ -1,51 +1,50 @@
...
+
+- **Real-time HTAP scenarios**
+
+    With the fast growth of 5G, Internet of Things, and artificial intelligence, the generated data for a company keeps increasing tremendously, reaching a scale of hundreds of TB (terabytes) or even the PB level. The traditional solution is to process online transactional applications using an OLTP database, use an ETL (Extract, Transform, Load) tool to replicate the data into an OLAP database for data analysis. This solution has the disadvantage of high storage cost. TiDB introduces the TiFlash column storage engine in v4.0, which combines with the TiKV row storage engine to build TiDB into a true HTAP database. With a small amount of extra storage cost, you can handle both online transactional processing and real-time data analysis in the same system, which can greatly save the cost.
```
Comment (ran-huang) [inline overview.md:46]:
```suggestion
    With the fast growth of 5G, Internet of Things, and artificial intelligence, the generated data for a company keeps increasing tremendously, reaching a scale of hundreds of TB (terabytes) or even the PB level. The traditional solution is to process online transactional applications using an OLTP database and use an ETL (Extract, Transform, Load) tool to replicate the data into an OLAP database for data analysis. This solution has the disadvantage of high storage cost. TiDB introduces the TiFlash columnar storage engine in v4.0, which combines with the TiKV row-based storage engine to build TiDB into a true HTAP database. With a small amount of extra storage cost, you can handle both online transactional processing and real-time data analysis in the same system, which can greatly save the cost.
```

File: overview.md [hunk 3]
```diff
@@ -1,51 +1,50 @@
...
+summary: Learn about the key features and usage scenarios of TiDB.
 category: introduction
-aliases: ['/docs/dev/overview/']
+aliases: ['/docs/dev/key-features/','/tidb/dev/key-features','/docs/dev/overview/']
 ---
 
 # TiDB Introduction
 
-[TiDB](https://github.com/pingcap/tidb) ("Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability.
+[TiDB](https://github.com/pingcap/tidb) ("Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability. The goal of TiDB is to provide users with a one-stop database solution -- OLTP (Online Transactional Processing), OLAP (Online Analytical Processing), and HTAP services. TiDB is suitable for various use cases that require high availability and strong consistency with large-scale data.
```
[the remark this answers is not in the document]
Comment (lilin90) [reply overview.md:10]: @ran-huang Thanks for pointing out. I think it should be `em` dash.

**Comment by lilin90** [comment]
@ran-huang All comments addressed. PTAL again~ Thanks!

File: overview.md [hunk 4]
```diff
@@ -1,51 +1,50 @@
...
 
-## Community provided blog posts & tutorials
+- **Cloud-native distributed database**
 
-The following list collects deployment guides and tutorials from the community. The content is subject to change by the contributors.
+    TiDB is a distributed database designed for the cloud—public cloud, private cloud, and hybrid cloud. [TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/tidb-operator-overview) automates the deployment on various clouds. [TiDB Cloud](https://pingcap.com/products/tidbcloud) (Beta), the fully-managed TiDB service, is the easiest, most economical, and most resilient way to unlock the full power of [TiDB in the cloud](https://docs.pingcap.com/tidbcloud/beta), allowing you to deploy and run TiDB clusters with just a few clicks.
```
Comment (lilin90) [inline overview.md:28]: I added TiDB Cloud to the feature description. @iamxy @tennix PTAL

Comment (iamxy) [reply overview.md]: TiDB Cloud is a commercial service offering by PingCAP, not a feature of TiDB. But to put it another way, thanks to TiDB's cloud-native, we successfully launched the TiDB Cloud.

Comment (iamxy) [reply overview.md]: TiDB is a distributed database designed for the cloud, providing flexible scalability, reliability and security on the cloud platform. Users can elastically scale TiDB to meet the requirements of their changing workload. In TiDB, each piece of data has 3 replicas at least, which can be scheduled in different cloud availability zones to tolerate the outage of a whole data center. [TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/tidb-operator-overview)  helps manage TiDB on Kubernetes and automates tasks related to operating TiDB cluster, which makes TiDB easier to deploy on any cloud that provides managed Kubernetes. [TiDB Cloud](https://pingcap.com/products/tidbcloud) (Beta), the fully-managed TiDB service, is the easiest, most economical, and most resilient way to unlock the full power of [TiDB in the cloud](https://docs.pingcap.com/tidbcloud/beta), allowing you to deploy and run TiDB clusters with just a few clicks.

Comment (lilin90) [reply overview.md]: Got it.

File: overview.md [hunk 5]
```diff
@@ -1,51 +1,50 @@
...
 
-## Community provided blog posts & tutorials
+- **Cloud-native distributed database**
 
-The following list collects deployment guides and tutorials from the community. The content is subject to change by the contributors.
+    TiDB is a distributed database specifically designed for the cloud—public cloud, private cloud, and hybrid cloud. [TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/tidb-operator-overview) automates the deployment on various clouds. [TiDB Cloud](https://pingcap.com/products/tidbcloud) (Beta), the fully-managed TiDB service, is the easiest, most economical, and most resilient way to unlock the full power of [TiDB in the cloud](https://docs.pingcap.com/tidbcloud/beta), allowing you to deploy and run TiDB clusters with just a few clicks.
```
Comment (lilin90) [inline overview.md:28]:
```suggestion
    TiDB is a distributed database designed for the cloud, providing flexible scalability, reliability and security on the cloud platform. Users can elastically scale TiDB to meet the requirements of their changing workloads. In TiDB, each piece of data has 3 replicas at least, which can be scheduled in different cloud availability zones to tolerate the outage of a whole data center. [TiDB Operator](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/tidb-operator-overview)  helps manage TiDB on Kubernetes and automates tasks related to operating the TiDB cluster, which makes TiDB easier to deploy on any cloud that provides managed Kubernetes. [TiDB Cloud](https://pingcap.com/products/tidbcloud) (Beta), the fully-managed TiDB service, is the easiest, most economical, and most resilient way to unlock the full power of [TiDB in the cloud](https://docs.pingcap.com/tidbcloud/beta), allowing you to deploy and run TiDB clusters with just a few clicks.
```

File: basic-features.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
+title: TiDB Basic Features
+summary: Learn about the basic features of TiDB.
+category: introduction
```
Comment (yikeke) [inline basic-features.md:4]:
```suggestion
category: introduction
aliases: []
```
Add two aliases from old/new website here?

File: overview.md [hunk 7]
```diff
@@ -1,51 +1,50 @@
...
 
-- [How To Spin Up an HTAP Database in 5 Minutes with TiDB + TiSpark](https://pingcap.com/blog/how_to_spin_up_an_htap_database_in_5_minutes_with_tidb_tispark/)
-- [Developer install guide (single machine)](http://www.tocker.ca/this-blog-now-powered-by-wordpress-tidb.html)
-- [TiDB Best Practices](https://pingcap.com/blog/2017-07-24-tidbbestpractice/)
+- **Compatible with the MySQL 5.7 protocol and MySQL ecosystem**
 
-_Your contribution is also welcome! Feel free to open a [pull request](https://github.com/pingcap/docs/blob/master/overview.md) to add additional links._
+    TiDB is compatible with the MySQL 5.7 protocol, common features of MySQL, and the MySQL ecosystem. To migrate your applications to TiDB, you do not need to change a single line of code in many cases or only need to modify a small amount of code. In addition, TiDB provides a series of [data migration tools](https://docs.pingcap.com/tidb/v4.0/migration-overview) to help migrate application data easily into TiDB.
```
Comment (yikeke) [inline overview.md:32]:
```suggestion
    TiDB is compatible with the MySQL 5.7 protocol, common features of MySQL, and the MySQL ecosystem. To migrate your applications to TiDB, you do not need to change a single line of code in many cases or only need to modify a small amount of code. In addition, TiDB provides a series of [data migration tools](/migration-overview.md) to help migrate application data easily into TiDB.
```

File: overview.md [hunk 8]
```diff
@@ -1,51 +1,50 @@
...
 
-    - [kind](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/get-started#create-a-kubernetes-cluster-using-kind)
-    - [Minikube](https://docs.pingcap.com/tidb-in-kubernetes/v1.1/get-started#create-a-kubernetes-cluster-using-minikube)
+- **Real-time HTAP**
 
-- [Binary Tarball Deployment](/production-deployment-from-binary-tarball.md): This guide describes how to deploy TiDB from a binary tarball in production. Guides for [development](/deploy-tidb-from-binary.md) and [testing](/test-deployment-from-binary-tarball.md) environments are also available.
+    TiDB provides two storage engines: [TiKV](https://tikv.org/), a row-based storage engine, and [TiFlash](https://docs.pingcap.com/tidb/v4.0/tiflash-overview), a columnar storage engine. TiFlash uses the Multi-Raft Learner protocol to replicate data from TiKV in real time, ensuring that the data between the TiKV row-based storage engine and the TiFlash columnar storage engine are consistent. TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
```
Comment (yikeke) [inline overview.md:24]:
```suggestion
    TiDB provides two storage engines: [TiKV](https://tikv.org/), a row-based storage engine, and [TiFlash](/tiflash/tiflash-overview.md), a columnar storage engine. TiFlash uses the Multi-Raft Learner protocol to replicate data from TiKV in real time, ensuring that the data between the TiKV row-based storage engine and the TiFlash columnar storage engine are consistent. TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
```

File: overview.md [hunk 9]
```diff
@@ -1,51 +1,50 @@
...
+
+- **Real-time HTAP scenarios**
+
+    With the fast growth of 5G, Internet of Things, and artificial intelligence, the generated data for a company keeps increasing tremendously, reaching a scale of hundreds of TB (terabytes) or even the PB level. The traditional solution is to process online transactional applications using an OLTP database and use an ETL (Extract, Transform, Load) tool to replicate the data into an OLAP database for data analysis. This solution has multiple disadvantages such as high storage cost and poor real-time performance. TiDB introduces the TiFlash columnar storage engine in v4.0, which combines with the TiKV row-based storage engine to build TiDB into a true HTAP database. With a small amount of extra storage cost, you can handle both online transactional processing and real-time data analysis in the same system, which can greatly save the cost.
```
Comment (yikeke) [inline overview.md:46]:
```suggestion
    With the fast growth of 5G, Internet of Things, and artificial intelligence, the data generated by a company keeps increasing tremendously, reaching a scale of hundreds of TB (terabytes) or even the PB level. The traditional solution is to process online transactional applications using an OLTP database and use an ETL (Extract, Transform, Load) tool to replicate the data into an OLAP database for data analysis. This solution has multiple disadvantages such as high storage costs and poor real-time performance. TiDB introduces the TiFlash columnar storage engine in v4.0, which combines with the TiKV row-based storage engine to build TiDB as a true HTAP database. With a small amount of extra storage cost, you can handle both online transactional processing and real-time data analysis in the same system, which greatly saves the cost.
```

**Pull request merged**